api/tests/routes/backoffice_v3/collective_offer_templates_test.py

```diff
@@ -389,3 +390,76 @@ def test_get_reject_form_test(self, legit_user, authenticated_client):
         with assert_num_queries(2):  # session + user
             response = authenticated_client.get(form_url)
             assert response.status_code == 200
+
+
+class BatchCollectiveOfferTemplatesValidateTest(PostEndpointHelper):
+    endpoint = "backoffice_v3_web.collective_offer_template.batch_validate_collective_offer_templates"
+    needed_permission = perm_models.Permissions.FRAUD_ACTIONS
+
+    def test_batch_validate_collective_offer_templates(self, legit_user, authenticated_client):
+        collective_offer_templates = educational_factories.CollectiveOfferTemplateFactory.create_batch(
+            3, validation=OfferValidationStatus.PENDING
+        )
+        parameter_ids = ",".join(
+            str(collective_offer_template.id) for collective_offer_template in collective_offer_templates
+        )
+        response = self.post_to_endpoint(authenticated_client, form={"object_ids": parameter_ids})
+
+        assert response.status_code == 303
+
+        expected_url = url_for(
+            "backoffice_v3_web.collective_offer_template.list_collective_offer_templates", _external=True
+        )
+        assert response.location == expected_url
+
+        for collective_offer_template in collective_offer_templates:
+            db.session.refresh(collective_offer_template)
+            assert collective_offer_template.lastValidationDate.strftime("%d/%m/%Y") == datetime.date.today().strftime(
+                "%d/%m/%Y"
+            )
+            assert collective_offer_template.isActive is True
+            assert collective_offer_template.lastValidationType is OfferValidationType.MANUAL
+            assert collective_offer_template.validation is OfferValidationStatus.APPROVED
```

Comment:
On peut aussi vérifier qu'un mail est bien envoyé : 

`from pcapi.core.users import testing as external_testing`
puis des assertions sur `external_testing.sendinblue_requests`.
Par exemple, hors de la boucle :
`assert {req["email"] for req in external_testing.sendinblue_requests} == {cot.venue.bookingEmail for cot in collective_offer_templates}`


```

Reply:
J'ai fais le test, la liste de `external_testing.sendinblue_requests` est vide.

Reply:
J'ai rajouté un test sur le destinataire, et un test sur le template pour ajout/rejet offre collective/offre collective vitrine.